src/betterproto/templates/template.py.j2

```diff
@@ -1,13 +1,13 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# sources: {{ ', '.join(description.files) }}
+# sources: {{ ', '.join(description.input_filenames) }}
 # plugin: python-betterproto
 from dataclasses import dataclass
 {% if description.datetime_imports %}
-from datetime import {% for i in description.datetime_imports %}{{ i }}{% if not loop.last %}, {% endif %}{% endfor %}
+from datetime import {% for i in description.datetime_imports|sort %}{{ i }}{% if not loop.last %}, {% endif %}{% endfor %}
```

Comment:
Would it make sense to implement this using isort similar to the way we use black today? Obviously as part of another changeset and not within here.

Reply:
That definitely makes sense for a future PR. Previously sort was being used somewhere else, I just moved it.